app/src/main/res/values/strings.xml

```diff
@@ -79,19 +79,18 @@
     <string name="aiMenuRegenerate">Regenerate a response</string>
     <string name="aiMenuSeriousWriting">Serious writing</string>
     <string name="aiMenuShorten">Shorten</string>
-    <string name="aiPromptAnswer">Tell the assistant how it can help you reply to this email.</string>
+    <string name="aiPromptAnswer">Explain to Euria how to get help replying to this email.</string>
     <string name="aiPromptExample1">a thank-you email for a welcome gift</string>
     <string name="aiPromptExample2">an email confirming your hotel reservation</string>
     <string name="aiPromptExample3">a thank-you email for an invitation to an event</string>
     <string name="aiPromptExample4">an invitation to a company event</string>
     <string name="aiPromptExample5">an email requesting an appointment with an important customer</string>
-    <string name="aiPromptGenerationLoader">The AI writing assistant is composing…</string>
-    <string name="aiPromptPlaceholder">Tell the AI writing assistant what you want to write. For example, \"%s\".</string>
-    <string name="aiPromptTag">Beta</string>
-    <string name="aiPromptTitle">The AI writing assistant</string>
+    <string name="aiPromptGenerationLoader">Euria is writing…</string>
+    <string name="aiPromptPlaceholder">Tell Euria what you want to write. For example \"%s\".</string>
+    <string name="aiPromptTitle">Euria</string>
     <string name="aiPromptValidateButton">Generate</string>
     <string name="aiReplaceSubjectDescription">The subject will become:\n\n%s</string>
-    <string name="aiReplaceSubjectTitle">Replace the current subject with the assistant’s proposal?</string>
+    <string name="aiReplaceSubjectTitle">Replace the current object with Euria’s proposal?</string>
```

Comment:
Keep the term "subject" instead of "object" which is the correct wording for an email's subject in english